openaerostruct/aerodynamics/eval_mtx.py

```diff
@@ -266,8 +266,7 @@ def setup(self):
 
             # It's worth the cs cost here because alpha is just a scalar
             self.declare_partials(vel_mtx_name, 'alpha', method='cs')
-
-            self.set_check_partial_options(wrt='*', method='cs')
+            self.set_check_partial_options(wrt='alpha', method='fd')
```

Comment:
What is the motivation here to switch to fd?

Reply:
It computes the partials using CS and checks them using CS which doesn’t make sense. The latest version of OpenMDAO raise an error in such case. I changed to check with FD to avoid the error (essentially it does nothing though)